packages/pods-core/src/builtin/AirtablePod.ts

```diff
@@ -402,28 +426,50 @@ export class AirtableUtils {
     records,
     engine,
     logger,
+    podId,
   }: {
     records: Records<FieldSet>;
     engine: DEngineClient;
     logger: DLogger;
+    podId?: string;
   }) {
+    if (!podId) return;
     const out = await Promise.all(
       records.map(async (ent) => {
         const airtableId = ent.id;
         const dendronId = ent.fields["DendronId"] as string;
         const note = engine.notes[dendronId] as NotePropsWithOptionalCustom;
-        const noteAirtableId = _.get(note.custom, "airtableId");
-        if (!noteAirtableId) {
-          const updatedNote = {
-            ...note,
-            custom: { ...note.custom, airtableId },
+        let pods: AirtablePodMetadata = _.get(note.custom, "pods");
+        // return if the note is already exported ie. update query is ran.
+        if (pods && pods.airtable && pods.airtable[podId]) return undefined;
+
+        // if this is the first time a pod metadata is added to the note, add airtable pod metadata under pods namespace
+        if (!pods) {
```

Comment:
Could we add some backwards compatibility here? If the old airtable frontmatter without the pods namespace exists, use the existing frontmatter. That way we don't have to immediately fix all notes with the doctor, just ones where we want to export to multiple targets.

Reply:
This method is only called when a new record is created. If `airtableId` is present in the note fronmatter, Airtable export v2 will update the following record and this method is not called.